packages/account/src/Components/forms/idv-form.tsx

```diff
@@ -0,0 +1,306 @@
+
+                    if (needs_additional_document) {
+                        return {
+                            id: key,
+                            text: new_display_name || display_name,
+                            additional: {
+                                display_name: document_data[key].additional?.display_name,
+                                format: document_data[key].additional?.format,
+                            },
+                            value: format,
+                            sample_image,
+                            example_format,
+                        };
+                    }
+                    return {
+                        id: key,
+                        text: new_display_name || display_name,
+                        value: format,
+                        sample_image,
+                        example_format,
+                    };
+                }),
+                IDV_NOT_APPLICABLE_OPTION,
+            ]);
+        }
+    }, [document_data, selected_country]);
+
+    const resetDocumentItemSelected = () => {
+        setFieldValue(
+            'document_type',
+            {
+                id: '',
+                text: '',
+                value: '',
+                example_format: '',
+                sample_image: '',
+            },
+            true
+        );
+    };
+
+    const getExampleFormat = (example_format: string) => {
+        return example_format ? localize('Example: ') + example_format : '';
+    };
+    const getDocument = (text: string) => {
+        return document_list.find(d => d.text === text);
+    };
+
+    const onKeyUp = (e: { target: HTMLInputElement }, document_name: string) => {
+        const { example_format } =
+            document_name === 'document_number' ? values.document_type : values.document_type.additional;
+        let current_input: string | null = null;
+        current_input = example_format.includes('-')
+            ? formatInput(example_format, current_input ?? e.target.value, '-')
+            : e.target.value;
+        setFieldValue(document_name, current_input, true);
+    };
+
+    return (
+        <React.Fragment>
+            <section className={classNames('idv-form', class_name)}>
+                <div className='details-form'>
+                    <div className='poi-form-on-signup__fields'>
+                        <div
+                            className={classNames('proof-of-identity__container', {
+                                'proof-of-identity__container--idv': hide_hint,
+                            })}
+                        >
+                            <div className={classNames('proof-of-identity__inner-container')}>
+                                <div className='proof-of-identity__fieldset-container'>
+                                    <fieldset className={classNames({ 'proof-of-identity__fieldset': !hide_hint })}>
+                                        <Field name='document'>
```

Comment:
A little confused as this Field has `name='document`, however we're overriding the name later on to `document_type`